presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcMetadata.java

```diff
@@ -354,8 +354,6 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
     public Map<String, ColumnHandle> getColumnHandles(ConnectorSession session, ConnectorTableHandle tableHandle)
     {
         JdbcTableHandle jdbcTableHandle = (JdbcTableHandle) tableHandle;
-        verify(!jdbcTableHandle.isSynthetic(), "Not a table reference: %s", jdbcTableHandle);
```

Comment:
Is it testable?

Reply:
I could write a "real" unit test. Then it would be just a inversion of implementation which I don't like.